docs/source/tutorials/object_detection.ipynb

```diff
@@ -202,7 +202,7 @@
    "id": "8f62da67",
    "metadata": {},
    "source": [
-    "`predictions` is a list where the predictions output by our model for the i-th image: `predictions[i]` is a list/array of shape `(K,)`. Here `K` is the number of classes in the dataset (same for every image) and `predictions[i][k]` is of shape `(M,5)`, where `M` is the number of bounding boxes predicted to contain objects of class `k` (in image i, differs between images). The five columns of `predictions[i][k]` correspond to ``[x1,y1,x2,y2,pred_prob]`` for each bounding box predicted by the model. `pred_prob` is the model confidence in its predicted label for this box. Since our dataset has `K = 5` classes, we have: `predictions[i].shape = (5,)`.\n",
+    "`predictions` is a list where the predictions output by our model for the i-th image: `predictions[i]` is a list/array of shape `(K,)`. Here `K` is the number of classes in the dataset (same for every image) and `predictions[i][k]` is of shape `(M,5)`, where `M` is the number of bounding boxes predicted to contain objects of class `k` (in image i, differs between images). The five columns of `predictions[i][k]` correspond to ``[x1,y1,x2,y2,pred_prob]`` format with respect to the y-coordinate axis for each bounding box predicted by the model. `pred_prob` is the model confidence in its predicted label for this box. Since our dataset has `K = 5` classes, we have: `predictions[i].shape = (5,)`.\n",
```

Comment:
```suggestion
    "`predictions` is a list where the predictions output by our model for the i-th image: `predictions[i]` is a list/array of shape `(K,)`. Here `K` is the number of classes in the dataset (same for every image) and `predictions[i][k]` is of shape `(M,5)`, where `M` is the number of bounding boxes predicted to contain objects of class `k` (in image i, differs between images). The five columns of `predictions[i][k]` correspond to ``[x1,y1,x2,y2,pred_prob]`` format (with respect to the y-coordinate axis) for each bounding box predicted by the model. `pred_prob` is the model confidence in its predicted label for this box. Since our dataset has `K = 5` classes, we have: `predictions[i].shape = (5,)`.\n",
```

Reply:
please make `y-coordinate axis` a hyperlink to MMDetection page that explains what this means.

For instance, I have no idea what this term means.